packages/manager/apps/ips/src/data/hooks/useExportToFile.ts

```diff
@@ -0,0 +1,66 @@
+import { useState, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNotifications } from '@ovh-ux/manager-react-components';
+import { aapi } from '@ovh-ux/manager-core-api';
+import { mkConfig, generateCsv, download } from 'export-to-csv';
+import { ListingContext } from '@/pages/listing/listingContext';
+import { ExportIpToCsvData } from '@/data/api';
+import { TRANSLATION_NAMESPACES } from '@/utils';
+
+export const useExportToCsv = () => {
+  const { t } = useTranslation([TRANSLATION_NAMESPACES.exportIpToCsv]);
+  const { apiFilter, ipToSearch } = useContext(ListingContext);
+  const { addSuccess, addError } = useNotifications();
+  const [isCSVLoading, setIsCSVLoading] = useState(false);
+
+  const handleExportToCsv = async () => {
+    setIsCSVLoading(true);
+    try {
+      const result = await aapi
+        .get('/ipsexportcsv', {
+          params: {
+            ...apiFilter,
+            serviceName: apiFilter['routedTo.serviceName'],
+            ip: ipToSearch,
```

Comment:
```suggestion
```

I think we will use the ip field from apiFilter directly

Reply:
The future is my next PR and I don't have this feature yet
Just remove it to avoid conflict